hud-data/broesel-light.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "broesel-light",
+  "author": "yawn",
+  "contributors": [
+    "Broesel",
+    "Hypnootize",
+    "Joe"
+  ],
+  "releaseDate": "2025-11-04",
+  "parent": "broesel-old",
+  "distinguishedFromParent": true,
+  "resources": [
+    "https://imgur.com/a/QIxUBS7"
```

Comment:
You must upload an image to the hud-resources/broesel-light folder and list out its name here, imgur links only go under the album like you already did.